test/test_server.js

```diff
@@ -58,6 +58,12 @@ TestServer.start = function (config, printLogs) {
 }
 
 TestServer.prototype.start = function () {
+  if (this.config.lastAccessTimeUpdates.enabled) {
+    // HACK: Force-enable devices.lastAccessTime for test/remote/device_tests.js
+    process.env.LASTACCESSTIME_UPDATES_ENABLED = 'true'
+    process.env.LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES = '.*'
+    process.env.LASTACCESSTIME_UPDATES_SAMPLE_RATE = '1'
```

Comment:
I encountered a similar issue when testing sign-in confirmation remote tests and all the different configurations. You should be able to move this logic to the `remote/device_tests.js` file, right before the server starts, [ex.](https://github.com/mozilla/fxa-auth-server/blob/master/test/remote/account_signin_verification_tests.js#L19). I am not sure which way is better or less hacky though.